contents/node/nodejs.md

```diff
@@ -0,0 +1,208 @@
+# Introduction to Node.js
+
+Author: Rachael Sim
+
+# Overview
+
+_This chapter assumes that the reader is familiar with JavaScript and asynchronous programming. If you are not familiar with asynchronous programming, a good resource to checkout is the [asynchronous programming section of the You Don't Know JS guide ](https://github.com/getify/You-Dont-Know-JS/blob/master/async%20&%20performance/README.md#you-dont-know-js-async--performance) as asynchronous programming is key in Node._
+
+*Node.js* is a JavaScript runtime built on Chrome’s V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.
+
+# Benefits
+
+Why should you use Node?
+
+## Benefit 1: Node.js is single-threaded
```

Comment:
May be rephrase to sound like a benefit right away? e.g., `No threading overhead`?
Alternatively, combine 1 and 2 `Better performance without the costs of multi-threading`.